source/_docs/blueprint/selectors.markdown

```diff
@@ -1005,6 +1005,13 @@ mode:
     will use a `dropdown` input.
   type: string
   required: false
+reorder:
+  description: >
+    When multiple is `true`, setting reorder to `true` will give each selected 
+    item a grabbable handle, which allows items to be reordered by drag and drop.
+  type: boolean
+  required: false
+  default: false
```

Comment:
This seems non-existing in the backend:

![CleanShot 2023-12-23 at 11 27 33@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/010d29d0-6e3d-40f7-8e81-7fee89d94853)

Reply:
I see, I didn't realize the backend had types for this. It does already exist in the frontend and we use it in the frontend for some select selectors.

So I guess it's just a choice of if we want to enable this feature for users, we would need to add the boolean flag there in the class definition.

Reply:
🤷 at this point, this make this PR invalid, as without the backend this won't fly.